components/card/card.js

```diff
@@ -1 +1,27 @@
-export function createCharacterCard() {}
+export function createCharacterCard(character) {
+  const newCard = document.createElement("li");
+  newCard.classList.add("card");
+
+  newCard.innerHTML = `
+      <div class="card__image-container">
+        <img
+          class="card__image"
+          src="${character.results.image}"
+          alt="${character.results.name}"
+        />
+        <div class="card__image-gradient"></div>
+      </div>
+      <div class="card__content">
+        <h2 class="card__title">${character.results.name}</h2>
+        <dl class="card__info">
+          <dt class="card__info-title">Status</dt>
+          <dd class="card__info-description">${character.results.status}</dd>
+          <dt class="card__info-title">Type</dt>
+          <dd class="card__info-description">${character.results.type}</dd>
+          <dt class="card__info-title">Occurrences</dt>
+          <dd class="card__info-description">${character.results.occurrences}</dd>
```

Comment:
Please, have a look at the API documentation again. Under `results.occurrences` you won't find the data you are looking for. You want to know in how many episodes the character appears. You will receive an array with all episodes from the API and need to determine the length of the array to get the number of episodes.

Also you do not need `results` here. If you have a look at a [single character object](https://rickandmortyapi.com/documentation/#get-a-single-character), you will see, that you can just use `character.name`, `character.type` etc.

Reply:
Hi Andrea, we resolved the issues. Could you have another look on our code to see if we did it right?